static/js/mixxx.js

```diff
@@ -0,0 +1,32 @@
+var OSName = getOSName();
+
+function getOSName() {
+  var OSName="your OS";
+  if (navigator.appVersion.indexOf("Win")!=-1) OSName="Windows";
```

Comment:
Let's use the same style we use for JS in controller mappings:
```suggestion
  if (navigator.appVersion.indexOf("Win") != -1) {
    OSName="Windows";
  }
```